workshops/dodge/README.md

```diff
@@ -4,7 +4,7 @@ Short link to this workshop: https://workshops.hackclub.com/dodge
 
 ---
 
-In this workshop we're going to build a simple yet addictive game using JavaScript. Previously when we've done web development projects, we've used HTML elements and things of that ilk to build what we see in the browser. Today we're going to do things a little differently and use some new tools-- JavaScript, along with two great libraries called p5.js and p5.play, to build our game.
+In this workshop, we're going to build a simple, yet addictive game using JavaScript. Previously when we've done web development, we've used HTML elements and things of that ilk to build what we see in the browser. Today we're going to do things a little differently and use a new coding language, JavaScript, along with two great libraries called p5.js and p5.play, to build our game.
```

Comment:
ey yo if you're gonna get down with commas you gotta put one after "yet addictive" too